priv/templates/boot_win.eex

```diff
@@ -493,12 +504,12 @@ if exists "%rel_dir%\vm.args.bak" do (
 @echo Extra Opts          %erl_opts%
 @echo.
 @echo Hooks:
-@for %%i in (%hook_dir%\*.d\*.bat) do (
```

Comment:
You're right, I didn't use hooks so I didn't notice I broke it (I just changed it to match the commands one as that one didn't work)